packages/serverpod/lib/src/server/session.dart

```diff
@@ -323,7 +323,7 @@ class MethodCallSession extends Session {
     required String method,
     required this.queryParameters,
     required super.authenticationKey,
-    super.enableLogging = true,
+    super.enableLogging,
```

Comment:
Question: This changes the default behavior since the variable is set to false now in the base session. Does this really work? The bool here is intended as an override for all other logging settings so that you are able to force turnoff all logging regardless of other settings.

This is mostly used with and important for the internal session.

Reply:
Reverted this change: [cec5fad](https://github.com/serverpod/serverpod/pull/2676/commits/cec5fad3d1dececc42a205ca6662da6c4342f464)